momento-redis/scalar_test.go

```diff
@@ -0,0 +1,363 @@
+package momento_redis_test
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"time"
+
+	"github.com/momentohq/client-sdk-go/auth"
+	"github.com/momentohq/client-sdk-go/config"
+	"github.com/momentohq/client-sdk-go/momento"
+	momentoredis "github.com/momentohq/momento-go-redis-client/momento-redis"
+	. "github.com/momentohq/momento-go-redis-client/momento-redis/test_helpers"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/redis/go-redis/v9"
+)
+
+var _ = Describe("Scalar methods", func() {
+	sContext := NewSharedContext()
+	BeforeEach(func() {
+		cacheName := "default_cache"
```

Comment:
You probably want to use a UUID here to prevent collisions if tests are run in parallel., e.g.: 

`fmt.Sprintf("golang-redis-%s", uuid.NewString())`